README.md

```diff
@@ -156,6 +156,6 @@ Graphql\init(new \GraphQL\Schema(['query' => $query(), 'mutation' => $mutation()
 ---
 More action here: [siler-examples](https://github.com/leocavalcante/siler-examples)
 
-⚠️️ **This is a work in progress, API may change** 🚧<br>
```

Comment:
Thank you very much for your corrections.
Can I ask you only to maintain this break lines? Without them the footer get one-lined.
Cheers.